README.md

```diff
@@ -1,3 +1,5 @@
+> **NOTE:** Kibana now includes [its own plugin generator](https://github.com/elastic/kibana/tree/master/packages/kbn-plugin-generator). You should use this to generate your plugin if you are targeting Kibana 6.3 or greater.
```

Comment:
Nice! Maybe also link to the part in contributing that explains how to use it all?

Reply:
Not sure what you mean. I linked to the plugin-generator readme because it starts off with `Setup`, talked about the kibana dev environment setup, and goes from there.

![image](https://user-images.githubusercontent.com/1329312/36127946-08a1d118-101d-11e8-87d4-9bdd7894376e.png)

Reply:
Oh, my bad, I didn't click through. Cool, seems GTG then.